webpack.config.js

```diff
@@ -0,0 +1,36 @@
+var webpack = require("webpack");
+var CopyWebpackPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+    entry: "./src/com/mendix/widget/carousel/Carousel.ts",
+    output: {
+        path: __dirname + "/dist/tmp",
```

Comment:
I tested, It could, though the documentation names as an absolute path. [output.path](https://github.com/webpack/docs/wiki/Configuration#outputpath)

To make it OS independent, it should actually be:
`path: path.resolve(__dirname, "dist", "tmp"),`